Sources/ManageMaintenance.php

```diff
@@ -1725,7 +1725,7 @@ function MaintainRecountPosts()
 	// Lets get a group of members and determine their post count (from the boards that have post count enabled of course).
 	$request = $smcFunc['db_query']('', '
 		SELECT /*!40001 SQL_NO_CACHE */ m.id_member, COUNT(m.id_member) AS posts
-		FROM ({db_prefix}messages AS m, {db_prefix}boards AS b)
+		FROM {db_prefix}messages AS m, {db_prefix}boards AS b
```

Comment:
Wait when was this done?

We don't use multiple from statements.  We use joins to bring in additional tables to make it clear what we are doing.  http://stackoverflow.com/questions/894490/sql-left-join-vs-multiple-tables-on-from-line